README.md

```diff
@@ -86,7 +98,14 @@ store = {
 33. [ ] update and add tests! (Abe)
 34. [x] update seedData on mLab (missing '.')
 35. [ ] css cleanup (ex. new item submit button not inline with cancel button) (Abe)(Linda)
+36. [x] general housekeeping (Abe)(Linda)
+37. [ ] second pass at general housekeeping (Abe)(Linda)
 
+#Future Features/Extensions
+1. Flag broken resources
```

Comment:
Can you provide an example of this? 
Would this be a situation where Alica A. borrowed from Bob b. a blender and it was broken or it broke in the process? (or Sell/Free usecase?) or is this to flag an invalid link?

Reply:
Yes, the Alice A > Bob B scenario - A borrowed item broke.  I wanted to include it as a potential future feature that you had brought up during our initial planning stage.  I will reword to make the note more clear.